core/templates/dev/head/app.constants.ajs.ts

```diff
@@ -0,0 +1,168 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Shared constants for the Oppia module.
```

Comment:
Please explain somewhere what's blocking the translation of these to Angular constants and include a TODO + issue to fix them as soon as doing so is feasible.

Reply:
We have #7092 for this very purpose. I'll do that in comments too.

Reply:
Added the comments!